hudi-metaserver/src/test/java/org/apache/hudi/metaserver/TestHoodieMetaServer.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.metaserver;
+
+import org.junit.jupiter.api.Test;
+
+public class TestHoodieMetaServer {
+  @Test
+  public void testEmbeddedServer() {
+    HoodieMetaServer.getMetadataStore();
```

Comment:
this UT is too barely minimum. can we assert some conditions here?

Reply:
add not null to check the server and server storage start successfully.